apps/elewa-group-website/src/app/app.component.html

```diff
@@ -1 +1,5 @@
-<elewa-group-home-page></elewa-group-home-page>
\ No newline at end of file
+<!-- <elewa-group-home-page></elewa-group-home-page>  -->
+<!-- 
+<elewa-group-card></elewa-group-card> -->
+
+<elewa-group-elewa-group-team-member></elewa-group-elewa-group-team-member>
```

Comment:
The tag name of this component is wrong,
Should be -> elewa-group-team-member not elewa-group-elewa-group-team-member

Reply:
fixed to elewa-group-team-member